cmd/controller-manager/app/controller_manager.go

```diff
@@ -303,7 +313,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		glog.V(1).Info("Starting shared informers")
 		informerFactory.Start(stop)
 	} else {
-		glog.V(1).Infof("Skipping starting service-catalog controller because servicecatalog/v1alpha1 is not available")
+		return fmt.Errorf("unable to start service-catalog controller: servicecatalog/v1alpha1 is not available")
```

Comment:
This is going to cause a death and (hopefully) restart, right? Is this the right thing to do? Is it better to do this vs try again later explicitly?